test/spec/Feature/Auth/AuthSpec.hs

```diff
@@ -99,11 +99,11 @@ spec = describe "authorization" $ do
   it "hides tables from users with invalid JWT" $ do
     let auth = authHeaderJWT "ey9zdGdyZXN0X3Rlc3RfYXV0aG9yIiwiaWQiOiJqZG9lIn0.y4vZuu1dDdwAl0-S00MCRWRYMlJ5YAMSir6Es6WtWx0"
     request methodGet "/authors_only" [auth] ""
-      `shouldRespondWith` [json| {"message":"JWSError (CompactDecodeError Invalid number of parts: Expected 3 parts; got 2)","code":"PGRST301","hint":null,"details":null} |]
+      `shouldRespondWith` [json| {"message":"BadCrypto","code":"PGRST301","hint":null,"details":null} |]
```

Comment:
This new message has less information but the previous one wasn't helpful for end users anyway. So I'm good with the change.